.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Thank you for contributing to curve! -->
+
+### What problem does this PR solve?
+
+Issue Number: close #xxx <!-- REMOVE this line if no issue to close -->
```

Comment:
<!-- REMOVE this line if no issue to close -->
this comment will not appear in markdown format...